postgres/standby_setup.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+
+
+if [ ! -s "$PGDATA/PG_VERSION" ]; then
+
+	pg_basebackup -d "host=$MASTER_NODE password=$REPLICATION_PASSWORD port=5432 user=$REPLICATION_USER" -D ${PGDATA} -vP
```

Comment:
add comment explaining what this does?
also, why you're not setting `set -e` until AFTER it?